proposals/umoci.md

```diff
@@ -0,0 +1,211 @@
+     list, ls  lists the set of tags in an OCI layout
+
+GLOBAL OPTIONS:
+   --verbose      alias for --log=info
+   --log value    set the log level (debug, info, [warn], error, fatal) (default: "warn")
+   --help, -h     show help
+   --version, -v  print the version
+```
+
+For a more detailed explanation of umoci, see the [project website's guide][umoci-guide].
+
+[umoci]: https://umo.ci/
+[umoci-guide]: https://umo.ci/quick-start/
+
+## Proposal ##
+Change the ownership of the existing umoci project from openSUSE:
+
+  https://github.com/openSUSE/umoci
+
+And move it inside the `opencontainers` organisation:
+
+  https://github.com/opencontainers/umoci
+
+The import paths will correspondingly be "github.com/opencontainers/umoci" (umoci does have some Go API users, but since the project will be renamed -- and GitHub will add a redirect -- there will be no significant downstream impact of the change).
+In the future we may opt to use vanity imports (such as "umo.ci/cmd/umoci").
+
+The project's domain "umo.ci" will also be transferred to the Linux Foundation so that it can be managed by someone other than the maintainers (though maintainers must maintain the necessary administrative access to maintain the website).
+
+### Initial Maintainers ###
+Initial maintainers of the umoci project would be:
+
+* Aleksa Sarai <cyphar@cyphar.com> (@cyphar)
+* Tycho Andersen <tycho@tycho.ws> (@tych0)
+* Vincent Batts <vbatts@hashbangbash.com> (@vbatts)
+
+### Code of Conduct ###
+This project would incorporate (by reference) the OCI [Code of Conduct][code-of-conduct].
+
+[code-of-conduct]: https://github.com/opencontainers/org/blob/master/CODE_OF_CONDUCT.md
+
+### Governance and Releases ###
+This project would initially incorporate the Governance and Release processes from [the OCI project template][oci-template], with the following modifications:
+
+ * Self-LGTMs are permitted, and maintainers may merge a PR that has only 1 LGTM (including their own) after 72 hours of giving an LGTM and pinging other maintainers.
+   This is necessary because the pool of maintainers is small enough that requiring 2 LGTMs from other maintainers effectively requires a unanimous vote on every change.
+   For the avoidance of doubt, this rule does **not** apply to the addition of maintainers to the project (which requires the usual qualified super-majority vote outlined in the [OCI template][oci-template]).
+
+   To avoid this rule being abused, or being kept in perpetuity, it shall have a mandatory sunset clause.
+   When 180 days (from umoci being included as an OCI project) have elapsed, or any new maintainers are added to the project (whichever happens first), this rule will expire unless a qualified super-majority of maintainers vote to keep the rule in place.
+   When reinstated, this rule shall have the same mandatory sunset clacse.
+
+   The motivation to make it easier to merge PRs is based on the maintainers' experience with the runc review process.
```

Comment:
Avoiding a deadlock process by simply merging doesn't feel right for a governed project.

Reply:
The idea was that if most maintainers lose interest (as happened in image-tools), then a single maintainer could keep the project active. But as I said above, I'll just switch this to self-LGTMs being allowed and we can always change this within the project if we feel it's needed.

Reply:
I get the problem in the larger scheme of things. The only concern I had with umochi was you were the only real maintainer, and from a single company. One of the main reasons for OCI was the vendor neutrality aspect, where no-one vendor could overly control a project. This was the same reason we added vbatts to oras, and josh had many of the commits. With vbatts and others as maintainers, that resolved my concern, but with a single/self lgtm, that brings back the same concern.

Reply:
Self-LGTM are different to only requiring a single LGTM. Self-LGTMs means that if a maintainer submits a PR they can LGTM their own change (but an additional LGTM from another maintainer is required). If we didn't permit self-LGTMs, each PR would have to effectively have a unanimous vote.

Reply:
On the topic of vendor neutrality and the set of initial maintainers, I do generally agree though I think there should be some implicit trust in the maintainers of any OCI project. While there are procedures for bad maintainers to be kicked out, we shouldn't assume that all maintainers are potentially malicious when reviewing these kinds of governance setups.

Don't get me wrong -- I am happy to add more maintainers to the umoci project. However, what I don't want to do is add people for the sake of adding them unless they actually could function as maintainers of the project (Vincent makes sense because he has both domain experience and has worked on similar projects in the past).

(As an aside, the SELinux proposal [was three-quarters Red Hat employees as maintainers](https://github.com/opencontainers/tob/blob/master/proposals/selinux.md#initial-maintainers).)

Reply:
The self and single LGTM was the blocking concern. Agree we should add folks that are truly interested and capable of engaging. It's just a balance.